caddyhttp/browse/browse.go

```diff
@@ -137,6 +139,15 @@ func (l byName) Swap(i, j int) { l.Items[i], l.Items[j] = l.Items[j], l.Items[i]
 
 // Treat upper and lower case equally
 func (l byName) Less(i, j int) bool {
+	return strings.ToLower(l.Items[i].Name) < strings.ToLower(l.Items[j].Name)
+}
```

Comment:
This still appears as having been modified, although it's restored code.

Please start your PR with the result of `git revert` as first commit.